security/compliance_operator/co-concepts/compliance-operator-understanding.adoc

```diff
@@ -15,8 +15,4 @@ The Compliance Operator is available for {op-system-first} deployments only.
 
 include::modules/compliance-profiles.adoc[leveloffset=+1]
 
-[id="additional-resources_compliance-operator-understanding"]
-[role="_additional-resources"]
-== Additional resources
-
-* xref:../../../security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc#compliance-operator-supported-profiles[Supported compliance profiles]
+include::modules/compliance-profile-types.adoc[leveloffset=+2]
```

Comment:
I noticed below validate error. Do we need to fix it?
```
Validating security/compliance_operator/co-concepts/compliance-operator-understanding.adoc ...
asciidoctor: WARNING: modules/compliance-profile-types.adoc: line 7: section title out of sequence: expected level 2, got level 3
```

Reply:
Hi Xiaojie - I fixed it. it was supposed to be an H1 header in the file instead of an H2 header.